packages/client/components/TeamPrompt/TeamPromptWorkDrawer.tsx

```diff
@@ -80,7 +94,14 @@ const TeamPromptWorkDrawer = (props: Props) => {
             {baseTabs.map((tab, idx) => (
               <Tab
                 key={tab.label}
-                onClick={() => setActiveIdx(idx)}
+                onClick={() => {
+                  SendClientSegmentEventMutation(atmosphere, 'Your Work Integration Clicked', {
+                    teamId: meeting.teamId,
+                    meetingId: meeting.id,
+                    integrationLabel: baseTabs[idx]?.label
```

Comment:
+1 Let's use `service` here to align naming as in, e.g., [`Integration Added`](https://github.com/ParabolInc/parabol/blob/00447aa9ac97e52449de23f3284e09cae9ff7bfe/packages/server/utils/analytics/analytics.ts#L387)